sdk/deviceupdate/azure-iot-deviceupdate/src/test/java/com/azure/iot/deviceupdate/DeploymentsClientTests.java

```diff
@@ -29,6 +28,10 @@ private DeploymentsAsyncClient createClient() {
         TokenCredential credentials;
         HttpClient httpClient;
         HttpPipelinePolicy recordingPolicy = null;
+        HttpPipeline httpPipeline;
+
+        HttpHeaders headers = new HttpHeaders().put("Accept", ContentType.APPLICATION_JSON);
+        AddHeadersPolicy addHeadersPolicy = new AddHeadersPolicy(headers);
```

Comment:
Can this be static? Also, I'm surprised AutoRest doesn't generate this as it does for other libraries

Reply:
It will be updated in a follow-up PR